stable/pomerium/values.yaml

```diff
@@ -0,0 +1,111 @@
+# For detailed explanation of each of the configuration settings see
+# https://www.pomerium.io/docs/config-reference.htmls
+
+# settings that are shared by all services
+config:
+  # secrets can be generated with `head -c32 /dev/urandom | base64`
+  sharedSecret: "LL3g/NoKkao7mTec2/wYSOZAki1VbR89q+kqbdFj0c4="
```

Comment:
@desimone it would be good if it was possible to specify a secret name as an alternative to the one generated by the template. For example I am a user of sealed-secrets and thus I generate the secrets myself.

Reply:
Will do. Could you point me to an example? I know some Charts use `randAlphaNum` (and the like) but unfortunately that's not quite the same as trying to get an exactly 32 byte random key.

Reply:
Sorry, maybe I wasn't clear. I meant to add a property `existingSecret` for example and if it is set to, say, `my-secret`, then secrets.yaml would generate nothing and the deployments would refer to `my-secret` instead of the one normally generated by `secrets.yaml`.
A bit like this: https://github.com/helm/charts/blob/master/stable/oauth2-proxy/values.yaml#L11